src/cadence/mainnet/getNFTs.cdc

```diff
@@ -921,16 +922,26 @@ pub fun getCaaPass(owner: PublicAccount, id: UInt64): NFTData? {
     let nft = col!.borrowCaaPass(id: id)
     if nft == nil { return nil }
 
+    let metadata: CaaPass.Metadata? = nft!.getMetadata()
+    if metadata == nil { return nil }
+
     return NFTData(
         contract: contract,
         id: nft!.id,
         uuid: nft!.uuid,
-        title: nil,
-        description: nil,
-        external_domain_view_url: nil,
+        title: metadata!.name,
+        description: metadata!.description,
+        external_domain_view_url: "https://thing.fund/",
         token_uri: nil,
-        media: [],
-        metadata: {},
+        media: [
+            NFTMedia(uri: "ipfs://".concat(metadata!.mediaHash), mimetype: metadata!.mediaType)
+        ],
+        metadata: {
+            "name": metadata!.name,
+            "description": metadata!.description,
+            "mediaType": metadata!.mediaType,
+            "mediaHash": metadata!.mediaHash
+        },
```

Comment:
Thanks for making these changes, could you also please update: https://github.com/alchemyplatform/alchemy-flow-contracts/blob/main/src/cadence/mainnet/getNFTIDs.cdc
https://github.com/alchemyplatform/alchemy-flow-contracts/blob/main/src/cadence/testnet/getNFTIDs.cdc
https://github.com/alchemyplatform/alchemy-flow-contracts/blob/main/src/cadence/testnet/getNFTs.cdc

Reply:
sure